recipes/sasview/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0d49040321cd7ada4df35c366e9759a05fae2c0f5082cf57f4eaa69f0ab16f04" %}  # openssl dgst -sha256 v5.0-beta.2.tar.gz
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: v5.0-beta.2.{{ file_ext }}
+  url: https://github.com/SasView/{{ name }}/archive/v5.0-beta.2.{{ file_ext }}
```

Would it make more sense to have:
```yaml
{% set version = "5.0-beta.2" %}
```
and then change this to:
```suggestion
  url: https://github.com/SasView/{{ name }}/archive/v{{ version }}.{{ file_ext }}
```

---

I am not sure we can use it as suggested. We already have a version which is based on the [merged PR](https://github.com/SasView/sasview/pull/206), and this is reflected in the version `v5.0_beta.2_8_g390494d7c` meaning it's not tagged yet.

Per @CJ-Wright's suggestion yesterday I used the tagged version's tarball with the patch corresponding to the PR's changes, however I'd still prefer to have the version I wrote above to not confuse potential users.

I'll commit a change with the `{% set tagged_version = "5.0-beta.2" %}` variable instead. Does it sound reasonable, @chrisburr?

---

See https://github.com/conda-forge/staged-recipes/pull/7839/commits/a53bafa6803a5cfca12a4f32a2ae8ccf1fea5769 for the change mentioned above.